site/triathlon/index.html

```diff
@@ -0,0 +1,82 @@
+<div class="row">
+    <!-- HERO/STARTBILD Vorlage -->
+    <div class="hero hero--image hero--light"
+        style="background: linear-gradient(0deg,#FCC509A3,#FCC509B8),url('https://images.unsplash.com/photo-1483721310020-03333e577078?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=2600&q=80&color=bw') center center / cover;">
+    </div>
+</div>
+<div class="row">
+    <div class="container">
+        <!-- SIDEBAR vorlage -->
+        <h1>Sidebar</h1>
+        <div class="main-content">
+            <div class="sidebar">
+                <div class="sticky">
+                    <div class="description">
+                        <h3>Auf dieser Seite</h3>
+                    </div>
+                    <ul>
+                        <li class="active">
+                            <a href="index.html">Home</a>
```

Comment:
Please don't use filenames. We later want to have clean urls like /triathlon (the server will find /triathlon/index.html).
Therefor also rename all files to index.html